plugin/web/components/common/Icon/icons.ts

```diff
@@ -4,6 +4,7 @@
 import { ReactComponent as Logo } from "./Icons/reearthLogo.svg";
 import { ReactComponent as Trash } from "./Icons/trash.svg";
 import { ReactComponent as PlateauLogo } from "./Icons/plateauLogo.svg";
+import { ReactComponent as PlateauLogoPrt } from "./Icons/plateauLogoPrt.svg";
```

Comment:
I can really understand what `Prt` stands for. If its not a long word, can you use that instead because I can't be sure what the difference between this and the normal PlateauLogo is.

Reply:
Prt is part of the logo , I 'll make it part.

Reply:
[pr-fix1](https://github.com/eukarya-inc/reearth-plateauview/pull/30/commits/4368dbe8b4bb396983a7cc21a87dc8497771a482)